src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -691,6 +697,10 @@ namespace cryptonote
     if (block_sync_size > BLOCKS_SYNCHRONIZING_MAX_COUNT)
       MERROR("Error --block-sync-size cannot be greater than " << BLOCKS_SYNCHRONIZING_MAX_COUNT);
```

Comment:
this is just a note / comment. Unrelated to this pr

L698 sets the limit to 2048, but its impossible to sync over 100 blocks because of this: https://github.com/monero-project/monero/blob/a1dc85c5373a30f14aaf7dcfdd95f5a7375d3623/src%2Fcryptonote_protocol%2Fcryptonote_protocol_handler.h#L58-L59